roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -28,6 +28,7 @@ openshift_logging_mux_journal_source: "{{ openshift_hosted_logging_journal_sourc
 openshift_logging_mux_journal_read_from_head: "{{ openshift_hosted_logging_journal_read_from_head | default('') }}"
 
 openshift_logging_mux_allow_external: False
+openshift_logging_use_mux: "{{ openshift_logging_mux_allow_external | default(False) }}"
```

Comment:
@richm, so if the user has mistakenly set `openshift_logging_mux_allow_external` to `true` (perhaps not realizing this behavior), and sets `openshift_logging_use_mux` to `false`, what will happen?